public/app/plugins/datasource/prometheus/configuration/PromSettings.tsx

```diff
@@ -304,6 +312,24 @@ export const PromSettings = (props: Props) => {
             />
           </div>
         </div>
+        <div className="gf-form-inline">
```

Comment:
this is still very rough

Reply:
Once this gets out of the proof of concept stage I'll review the proposed UI changes with UX